pages/common/pip3.md

```diff
@@ -3,25 +3,21 @@
 > Python package manager.
 > More information: <https://pip.pypa.io>.
 
-- Find available packages:
-
-`pip3 search {{package_name}}`
-
```

Comment:
What's your rationale for that?

Reply:
@acuteenvy informed in a previous review comment that this behavior has been removed from pip.

Reply:
https://github.com/tldr-pages/tldr/pull/10492#discussion_r1289021279

Reply:
What a weird thing of PyPA to do